internal/dataplane/sendconfig/strategy.go

```diff
@@ -14,8 +14,9 @@ import (
 
 // ContentWithHash encapsulates file.Content along with its precalculated hash.
 type ContentWithHash struct {
-	Content *file.Content
-	Hash    []byte
+	Content        *file.Content
+	CustomEntities map[string][]map[string]interface{}
```

Comment:
The underlying type of `CustomEntities` is ambiguous. Can we add a separate type (could be an alias) for that and add a godoc explaining what is actually stored in it? For instance what is the (first) map here for? What does slice store? etc.

Reply:
Added an alias.